web/src/app/admin/AdminConfig.js

```diff
@@ -127,6 +137,26 @@ export default class AdminConfig extends React.PureComponent {
                           value: f.value,
                         }))}
                     />
+                    {hintGroups[groupID] &&
+                      hintGroups[groupID].map(h => (
+                        <TextField
+                          key={h.id}
+                          readOnly
+                          label={hintName(h.id)}
+                          value={h.value}
+                          variant='filled'
+                          margin='none'
+                          InputProps={{
+                            margin: 'dense',
```

Comment:
Can we remove dense here? It looks very cluttered with this set, especially when multiple links are displayed.